### PR TITLE
 DAOS-2239 tests: Adjusting ior tests to new DAOS driver for ior

### PR DESCRIPTION
**Pull request by saurabhtandan**
Renaming multipleclients.py to four_servers.py
and modifying it. Renaming multipleclients.yaml
to four_servers.yaml and modifying it.
Also modified eight_server.py, eight_server.yaml,
nvme_io.py and nvme_io.yaml to adjust to new 
IOR daos api

### CODE REVIEW

File: src/tests/ftest/io/four_servers.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            - boro-H
+            - boro-I
+            - boro-J
+            - boro-K
+            - boro-L
+server_config:
+    name: daos_server
+pool:
+    createmode:
+        mode_RW:
```
Comment (ravalsam) [inline src/tests/ftest/io/four_servers.yaml]: Can be removed

File: src/tests/ftest/io/nvme_io.py [hunk 2]
```diff
@@ -51,6 +52,7 @@ class NvmeIo(avocado.Test):
     def setUp(self):
```
Comment (ravalsam) [inline src/tests/ftest/io/nvme_io.py]: I am also working on changing this as per of PR#400 (Code updated with apricot class), so we might have to rebase based on what ever patch land first

File: src/tests/ftest/io/eight_servers.py [hunk 3]
```diff
@@ -26,112 +26,58 @@
...
+class EightServers(TestWithServers):
     """
     Test class Description: Runs IOR with 8 servers.
+    :avocado: recursive
     """
 
-    def __init__(self, *args, **kwargs):
-
-        super(EightServers, self).__init__(*args, **kwargs)
+    def setUp(self):
```
Comment (mjean308) [inline src/tests/ftest/io/eight_servers.py]: I was just reviewing the SetUp in apricot and it looks like you don't need the setUp at all ...
The client hostfiles are also created in setUp in TestWithServer class

Comment (saurabhtandan) [reply src/tests/ftest/io/eight_servers.py]: Done

File: src/tests/ftest/io/eight_servers.py [hunk 4]
```diff
@@ -26,112 +26,52 @@
 import os
 import json
```
Comment (jolivier23) [inline src/tests/ftest/io/eight_servers.py]: remove json now that it's unused?

Comment (saurabhtandan) [reply src/tests/ftest/io/eight_servers.py]: Done

File: src/tests/ftest/io/eight_servers.yaml [hunk 5]
```diff
@@ -24,49 +30,41 @@ pool:
...
-            blocksize: '480m'
+        np_1:
+            np: 1
+        np_64:
+            np: 64
+        np_128:
+            np: 128
     iteration:
-        iter: 1
+        iter: 5
```
[the remark this answers is not in the document]
Comment (saurabhtandan) [reply src/tests/ftest/io/eight_servers.yaml]: time out was increased.

**Pull request merged**